account_cutoff_accrual_base/models/account_cutoff.py

```diff
@@ -13,28 +12,14 @@ class AccountCutOff(models.Model):
     @api.model
     def _default_cutoff_account_id(self):
         account_id = super(AccountCutOff, self)._default_cutoff_account_id()
-        type = self.env.context.get('default_type')
+        cutoff_type = self.env.context.get('default_cutoff_type')
         company = self.env.user.company_id
-        if type == 'accrued_expense':
+        if cutoff_type == 'accrued_expense':
             account_id = company.default_accrued_expense_account_id.id or False
-        elif type == 'accrued_revenue':
+        elif cutoff_type == 'accrued_revenue':
             account_id = company.default_accrued_revenue_account_id.id or False
         return account_id
 
-    @api.model
-    def _get_default_journal(self):
-        journal = super(AccountCutOff, self)._get_default_journal()
-        cutoff_type = self.env.context.get('default_type')
-        company = self.env.user.company_id
-        default_journal = company.default_cutoff_journal_id
-        if cutoff_type == 'accrued_expense':
-            journal = company.default_accrual_expense_journal_id or\
-                default_journal
-        elif cutoff_type == 'accrued_revenue':
-            journal = company.default_accrual_revenue_journal_id or\
-                default_journal
-        return journal
```

Comment:
Hi @alexis-via this method and associated default values are used in particular in `account_invoice_accrual`.
Can we preserve them? 

cc/ @sbejaoui

Reply:
Hi @alexis-via,
I created a [PR](https://github.com/akretion/account-closing/pull/2) to add missing fields.
Can you check it please.

Reply:
I wanted to simplify all these additionnal settings and keep the ones that are really used by a majority of users... but your collegues convinced me :-(